data_steward/cdr_cleaner/cleaning_rules/cancer_concept_suppression.py

```diff
@@ -23,14 +23,14 @@
 
 LOGGER = logging.getLogger(__name__)
 
-SUPPRESSION_RULE_CONCEPT_TABLE = 'cancer_condition_concepts'
+SUPPRESSION_RULE_CONCEPT_TABLE = 'cancer_condition_concepts_without_other_cancer'
 
 CANCER_CONCEPT_QUERY = JINJA_ENV.from_string("""
 CREATE OR REPLACE TABLE `{{project_id}}.{{sandbox_id}}.{{concept_suppression_lookup_table}}` AS
 SELECT  c.*
 FROM `{{project_id}}.{{dataset_id}}.concept` c
 WHERE REGEXP_CONTAINS(c.concept_code, 
-    r'(History_WhichConditions)|(Condition_OtherCancer)|(History_AdditionalDiagnosis)|(OutsideTravel6MonthWhere)'
+    r'(History_WhichConditions)|(History_AdditionalDiagnosis)|(OutsideTravel6MonthWhere)'
 )
```

Comment:
The ticket mentions a PPI filter as well. I tried to look for non-PPI concepts but couldn't find any, so this still works, but might be good to filter in case non-PPI concepts in the future. 
```suggestion
) AND vocabulary_id = 'PPI'
```